firefox-ios/Client/Frontend/Autofill/Address/Edit/EditAddressLocalization.swift

```diff
@@ -79,42 +79,41 @@ struct EditAddressLocalization: Encodable {
         case autofillSaveButton = "autofill-save-button"
     }
 
-    // TODO: FXIOS-9100 Change the correct strings after UX finalize copyright
     static let editAddressLocalizationIDs = EditAddressLocalization(
-        autofillAddressGivenName: "Given Name",
-        autofillAddressAdditionalName: "Additional Name",
-        autofillAddressFamilyName: "Family Name",
-        autofillAddressName: "Full Name",
-        autofillAddressOrganization: "Organization",
-        autofillAddressStreet: "Street Address",
-        autofillAddressState: "State",
-        autofillAddressProvince: "Province",
-        autofillAddressCity: "City",
-        autofillAddressCountry: "Country",
-        autofillAddressZip: "ZIP Code",
-        autofillAddressPostalCode: "Postal Code",
-        autofillAddressEmail: "Email Address",
-        autofillAddressTel: "Telephone Number",
-        autofillEditAddressTitle: "Edit Address",
-        autofillAddressNeighborhood: "Neighborhood",
-        autofillAddressVillageTownship: "Village/Township",
-        autofillAddressIsland: "Island",
-        autofillAddressTownland: "Townland",
-        autofillAddressDistrict: "District",
-        autofillAddressCounty: "County",
-        autofillAddressPostTown: "Post Town",
-        autofillAddressSuburb: "Suburb",
-        autofillAddressParish: "Parish",
-        autofillAddressPrefecture: "Prefecture",
-        autofillAddressArea: "Area",
-        autofillAddressDoSi: "District or Sub-island",
-        autofillAddressDepartment: "Department",
-        autofillAddressEmirate: "Emirate",
-        autofillAddressOblast: "Oblast",
-        autofillAddressPin: "Pincode",
-        autofillAddressEircode: "Eircode",
-        autofillAddressCountryOnly: "Country Only",
-        autofillCancelButton: "Cancel",
-        autofillSaveButton: "Save"
+        autofillAddressGivenName: String.Addresses.Settings.Edit.AutofillAddressName,
+        autofillAddressAdditionalName: String.Addresses.Settings.Edit.AutofillAddressName,
+        autofillAddressFamilyName: String.Addresses.Settings.Edit.AutofillAddressName,
+        autofillAddressName: String.Addresses.Settings.Edit.AutofillAddressName,
```

Comment:
Same for all? 🤔

Reply:
Good catch. We should fix this probably but this wouldn't break anything AFAICT. Since we now exclusively only use a single name field. I will open a patch on Desktop to remove those unused strings.